### ISSUE TITLE
Account settings never shows up 

### ISSUE THREAD

**Issue by Sivnarayan-Ka** [microsoft/vscode#116586]
Issue Type: <b>Bug</b>

I have downloaded the zip file for visual studio available on the microsoft website official . so why cannot i login to  visual studio . The sign in option doesnot show up anywhere .Neither in file menu nor in the accounts icon on the side bar. Can you please help me solve the issue.

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 6.3.9600

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4700MQ CPU @ 2.40GHz (8 x 2394)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.73GB (4.38GB free)|
|Process Argv|--crash-reporter-id 9b28d469-4090-437b-b49b-fe887aaff61f|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by RMacfarlane** [comment]
@Sivnarayan-Ka Are you trying to sign in to settings sync? What does the accounts menu in the side bar look like, can you include a screen shot?

**Comment by Sivnarayan-Ka** [comment]
Yes it shows only two things:
1) you are not signed in to any account(no options anywhere to sign in)
2) turn on sync settings (this is clickable ,clicking which shows msg "this
is preview feature ,read documentation before turning on".
There is no option for account settings also anywhere neither in menus or
in any side bars.Attaching screenshot for reference.

So I thought to drop this standalone version and went on to use the
community version setup from the Microsoft site which gave me an option to
login to an account on first startup/installation.But even in this also
when I tried to push and commit it showed success on VS code ide however
when cross verifying on within account there was not new commits shown.

Hence due to time constraints I dropped diagnosing the issue and have
started using cmd prompt old school way and committing to github.

Thanks,
Sivnarayan
